Source/cmake/FindGLibCompileResources.cmake

```
@@ -0,0 +1,84 @@
+pkg_get_variable(GLIB2_PREFIX glib-2.0 prefix)
+find_program(
+    GLIB_COMPILE_RESOURCES_EXE
+    NAMES glib-compile-resources
+    HINTS ${GLIB2_PREFIX}
+    REQUIRED
+)
+execute_process(
+    COMMAND ${GLIB_COMPILE_RESOURCES_EXE} --version
+    OUTPUT_VARIABLE glib_compile_resources_version
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+message(STATUS "Found glib-compile-resources (required): "
+    ${GLIB_COMPILE_RESOURCES_EXE}
+    " (${glib_compile_resources_version})")
+if (${glib_compile_resources_version} VERSION_LESS 2.77)
```

I would quote the expansion of the variable, just in case something bad happens during execution and it ends up being empty, and that way avoid a syntax error from CMake:

```suggestion
if ("${glib_compile_resources_version}" VERSION_LESS 2.77)
```